python/google/protobuf/internal/decoder.py

```diff
@@ -172,7 +172,10 @@ def SpecificDecoder(field_number, is_repeated, is_packed, key, new_default,
                       clear_if_default=False):
     if is_packed:
       local_DecodeVarint = _DecodeVarint
-      def DecodePackedField(buffer, pos, end, message, field_dict):
+      def DecodePackedField(
```

Comment:
I believe best practice in python is to just name unused variable `_`, without the `del` (and below)